example/spec/serial/DiscoverGranulesHttpSpec.js

```diff
@@ -183,15 +183,25 @@ describe('The Discover Granules workflow with http Protocol', () => {
     let ingestGranuleExecutionStatus;
     let syncGranuleLambdaOutput;
 
-    beforeAll(async () => {
+    beforeAll(() => {
       ingestGranuleWorkflowArn1 = queueGranulesOutput.payload.running[0];
       ingestGranuleWorkflowArn2 = queueGranulesOutput.payload.running[1];
       ingestGranuleWorkflowArn3 = queueGranulesOutput.payload.running[2];
-      console.log('\nwait for ingestGranuleWorkflow', ingestGranuleWorkflowArn1);
-      ingestGranuleExecutionStatus = await waitForCompletedExecution(ingestGranuleWorkflowArn1);
     });
 
-    it('executes successfully', () => {
+    it('executes successfully', async () => {
+      console.log('\nwait for ingestGranuleWorkflow', ingestGranuleWorkflowArn1);
+      ingestGranuleExecutionStatus = await waitForCompletedExecution(ingestGranuleWorkflowArn1);
```

Comment:
Why not wait for API on this execution as well?

Reply:
Or more importantly, this is already called in the after all: 

```
    await Promise.all(queueGranulesOutput.payload.running
      .map((arn) => waitForApiStatus(
        getExecution,
        {
          prefix: config.stackName,
          arn,
        },
        'completed'
      )));
```

Reply:
Good point, this was just a copy paste. I can update this!